src/sentry/static/sentry/app/views/organizationHealth/details.jsx

```diff
@@ -0,0 +1,72 @@
+import {Flex} from 'grid-emotion';
+import React from 'react';
+
+import {t} from 'app/locale';
+import LineChart from 'app/components/charts/lineChart';
+import PieChart from 'app/components/charts/pieChart';
+
+import HealthPanelChart from './styles/healthPanelChart';
+import HealthRequest from './util/healthRequest';
+import withHealth from './util/withHealth';
+
+class OrganizationHealthDetails extends React.Component {
```

Comment:
TBD If we can keep this as a generic component, there may be specific components for each "detail" view.